tests/__snapshots__/vento.test.ts.snap

```diff
@@ -130,7 +130,7 @@ snapshot[`build a site with vento 3`] = `
   <title>TÍTULO</title>
 </head>
 <body>
-  <h1>Título</h1>
+  <section class="container"><h1>Título</h1>
```

Comment:
`header` has gone.

Is there some conflicts between layouts `content` and comp's one?

```suggestion
  <section class="container"><header>Header</header><h1>Título</h1>
```